components/layout_2020/display_list/mod.rs

```diff
@@ -37,7 +37,12 @@ type ItemTag = (u64, u16);
 type HitInfo = Option<ItemTag>;
 
 pub struct DisplayListBuilder<'a> {
+    /// The current SpatialId and ClipId information for this `DisplayListBuilder`.
     current_space_and_clip: wr::SpaceAndClipInfo,
+
+    /// The id of the nearest ancestor reference frame for this `DisplayListBuilder`.
+    nearest_reference_frame: wr::SpatialId,
```

Comment:
What is a *reference frame* defined as? A quick Google search doesn't pull up that term in a CSS spec.

Reply:
The `reference frame` is a concept from WebRender. Essentially it is the thing that does transformations and starts new coordinate systems.